const problem = (x) => typeof(x) == 'number' ? x * 50 + 6  : "Error"